src/app/ui/external/home/home.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="flex flex-col items-center">
```

Comment:
this whole file is not doing anything, even the button is a deadend- clean this html

Reply:
Cleaned